src/engine/SmtCore.h

```diff
@@ -115,18 +127,9 @@ class SmtCore
 
 private:
     /*
-      A stack entry consists of the engine state before the split,
-      the active split, the alternative splits (in case of backtrack),
-      and also any implied splits that were discovered subsequently.
-    */
-    struct StackEntry
-    {
-    public:
-        PiecewiseLinearCaseSplit _activeSplit;
-        List<PiecewiseLinearCaseSplit> _impliedValidSplits;
-        List<PiecewiseLinearCaseSplit> _alternativeSplits;
-        EngineState *_engineState;
-    };
+      duplicate the StackEntry
+    */
+    StackEntry *duplicateStackEntry( const StackEntry &stackEntry );
```

Comment:
Looks like this should be a method of the StackEntry class

Reply:
Moved to StackEntry